test/end2end/scripts/python/Solver.py

```diff
@@ -691,7 +691,6 @@ def _getAMPLOptionsName(self):
         return "mosek"
 
     def __init__(self, exeName, timeout=None, nthreads=None, otherOptions=None):
-        stags = {ModelTags.continuous, ModelTags.integer, ModelTags.binary,
-                 ModelTags.quadratic
-                 }
+        stags = {ModelTags.continuous}
+        # ModelTags.integer, ModelTags.binary, ModelTags.quadratic
```

Comment:
As soon as integer variables are recognized by the driver, these tags should be enabled to allow the corresponding tests